lib/swaggerUI.js

```diff
@@ -23,18 +23,25 @@ function staticServe(hyper, req) {
         if (reqPath === '/index.html') {
             // Rewrite the HTML to use a query string
             const cfg = hyper.config;
+            const css = `
+                .topbar-wrapper .link img {
+                    display: none;
+                }
+
+                .topbar-wrapper .link::after {
+                    content: "${cfg.ui_name}";
+                }
+            `;
             body = body.toString()
                 .replace(/((?:src|href)=['"])/g, '$1?path=')
                 // Some self-promotion
-                .replace(/<a id="logo".*?<\/a>/,
-                    `<a id="logo" href="${cfg.ui_url}">${cfg.ui_name}</a>`)
+                .replace(/<\/style>/, `${css}\n  </style>`)
                 .replace(/<title>[^<]*<\/title>/, `<title>${cfg.ui_title}</title>`)
                 // Replace the default url with ours, switch off validation &
                 // limit the size of documents to apply syntax highlighting to
-                .replace(/docExpansion: "none"/, 'docExpansion: "list", ' +
-                    'validatorUrl: null, ' +
-                    'highlightSizeThreshold: 10000')
-                .replace(/ url: url,/, 'url: "?spec",');
+                .replace(/dom_id: '#swagger-ui'/, 'dom_id: "#swagger-ui", ' +
+                    'docExpansion: "list", validatorUrl: null, displayRequestDuration: true')
+                .replace(/"https:\/\/petstore.swagger.io\/v2\/swagger.json"/, `${req.headers.host}${req.uri}`);
```

Comment:
You can still just use `"?spec" here to have a relative reference.

Reply:
Done